sbnanaobj/StandardRecord/SRNuID.h

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRNuID.h
+// \brief   SRNuID object for Neutrino ID score features (MVA inputs). This SR code copied/based on other SR objects.
+// \author  $Author: howard@fnal.gov
+////////////////////////////////////////////////////////////////////////
+#ifndef SRNUID_H
+#define SRNUID_H
+
+namespace caf
+{
+  /// Slice Neutrino ID Score features (MVA inputs)
+  class SRNuID
+  {
+  public:
+    SRNuID();
+    virtual ~SRNuID();
+
+    float        NuNFinalStatePfos;        //!< NuNFinalStatePfos feature in NeutrinoID
+    unsigned int NuNHitsTotal;             //!< NuNHitsTotal feature in NeutrinoID
+    float        NuVertexY;                //!< NuVertexY feature in NeutrinoID
+    float        NuWeightedDirZ;           //!< NuWeightedDirZ feature in NeutrinoID
+    float        NuNSpacePointsInSphere;   //!< NuNSpacePointsInSphere feature in NeutrinoID
+    float        NuEigenRatioInSphere;     //!< NuEigenRatioInSphere feature in NeutrinoID
+    float        CRLongestTrackDirY;       //!< CRLongestTrackDirY feature in NeutrinoID
+    float        CRLongestTrackDeflection; //!< CRLongestTrackDeflection feature in NeutrinoID
+    float        CRFracHitsInLongestTrack; //!< CRFracHitsInLongestTrack feature in NeutrinoID
+    unsigned int CRNHitsMax;               //!< CRNHitsMax feature in NeutrinoID
```

Comment:
I'm going to try and continue my campaign of advocating for shorter all-lowercase names...
If we decide that's not experiment CAF style we should do that formally somehow and write it down. The argument is that short lowercase names makes it less likely you will have to check doxygen (which we don't even have...) just to remember how to spell something.
I realize that it's kind of tricky here without making a big mess, and that you're copying the names in the pandora data structures, so I don't have great suggestions.

Reply:
Are there prospects on us having a doxygen? I'd definitely like that. I find myself always opening a CAF file to examine the variables that it contains.

On the discussion of shorter versus longer names, I'm happy to go with whatever the convention is, though to me `crlongtrkdef` doesn't _necessarily_ seem more memorable to me than `CRLongestTrackDeflection`.

Reply:
doxygen has been discussed at an infrastructure meeting. Technically it's very easy to do, but hosting is a question. I have scripting for the dune doxygen, but I'm not willing to be responsible for it. I never figured out all the kcron business required to have a fermilab server automatically updated, so the dune doxygen only updates when someone bugs me. If we agreed to host at basically any university this would be easy again.